services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -197,12 +197,6 @@ testApproveLegalHoldDevice = do
         putEnabled tid LegalHoldEnabled
         requestDevice owner member tid !!! const 204 === statusCode
 
-        putEnabled tid LegalHoldDisabled
```

Comment:
This block is no longer necessary since Disabling LH nukes settings and all; which is tested elsewhere